src/ray/protobuf/event.proto

```diff
@@ -36,6 +36,11 @@ message Event {
     WARNING = 1;
     ERROR = 2;
     FATAL = 3;
+    // Ideally we would want these to be smaller numerically than INFO, but
+    // unfortunately the original rpc enum comes without such, and to be backward
+    // compatible we have to keep the same values for INFO.
+    DEBUG = 4;
+    TRACE = 5;
```

Comment:
DEBUG and TRACE is a bit close. Could we add a short guidance to tell what would be the recommended way of using them?